packages/core/src/utils/parseInvitation.ts

```diff
@@ -13,15 +10,11 @@ import { OutOfBandInvitation } from '../modules/oob/messages'
  * @returns OutOfBandInvitation
  */
 export const parseInvitationUrl = async (invitationUrl: string): Promise<OutOfBandInvitation> => {
```

Comment:
I think the flow right now is:
1. check inline oob invitation url
2. try to fetch shortened oob invitation url
3. check inline connection invitation url
4. try to fetch shortened connection invitation url

Could we maybe reorder this to:
1. check inline oob invitation url
2. check inline connection invitation url
3. try to fetch shortened oob invitation url
4. try to fetch shortened connection invitation url